src/plot_api/plot_schema.js

```diff
@@ -460,11 +460,22 @@ function getTraceAttributes(type) {
     // make 'type' the first attribute in the object
     attributes.type = null;
 
+
+    var copyBaseAttributes = extendDeepAll({}, baseAttributes),
+        copyModuleAttributes = extendDeepAll({}, _module.attributes);
```

Comment:
```js
var a = {},
    b = [];
```

have gone out of style in our codebase. It might be worth adding an `eslint` rule for this. From

https://eslint.org/docs/rules/one-var#initialized-and-uninitialized

looks like `"one-var": { "initialized": "never", "uninitialized": "consecutive" }` is exactly what most of the codebase does already.

Reply:
I fixed this in f168ddc and opened an issue (https://github.com/plotly/plotly.js/issues/3162) so I don't forget about adding an `eslint` rule for this.